tests/Fixer/Strict/StrictTypesFixerTest.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Strict;
+
+use PhpCsFixer\Test\AbstractFixerTestCase;
+
+/**
+ * @internal
+ */
+final class StrictTypesFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixCases()
+    {
+        return array(
+            array(
+                '<?php declare(strict_types=1);
+declare(ticks=1);
+declare(encoding=\'ISO-8859-1\');
+
+
+namespace A\B\C;
+class A {
+}',
+                '<?php
+declare(ticks=1);
+declare(encoding=\'ISO-8859-1\');
+declare(strict_types=1);
+
+namespace A\B\C;
+class A {
+}',
+            ),
+            array(
+                '<?php declare/* A b C*/(strict_types=1);
+//abc',
+                '<?php DECLARE/* A b C*/(strict_types=1);      //abc',
+            ),
+            array(
+                '<?php declare/* A b C*/(strict_types=1);',
+            ),
+            array(
+                '<?php declare(strict_types=1);
+/**/ /**/       ?>Test',
+                '<?php /**/ /**/ deClarE  (STRICT_TYPES=1)    ?>Test',
+            ),
+            array(
+                '<?php declare(strict_types=1);
+',
+                '<?php            DECLARE  (    strict_types=1   )   ;',
+            ),
+            array(
+                '<?php declare(strict_types=1);
+/**/
+                ',
+                '<?php
+                /**/
+                declare(strict_types=1);',
+            ),
+            array(
+                '<?php declare(strict_types=1);
+/**/ /**/ /**/
+                 /* abc */ ',
+                '<?php
+                /**/ /**/ /**/
+                declare /* abc */ (strict_types=1);',
+            ),
+            array(
+                '<?php declare(strict_types=1);
+                phpinfo();',
+                '<?php
+
+                phpinfo();',
+            ),
+            array(
+                '<?php declare(strict_types=1);
+/**
+ * Foo
+ */
+phpinfo();',
+                '<?php
+
+/**
+ * Foo
+ */
+phpinfo();',
+            ),
+            array(
+                '<?php declare(strict_types=1);
```

Comment:
default? so there is also other, non-default option? how could I configure it to have it on other line ?